src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -382,182 +370,180 @@ class ViewAllLogs extends Component {
-          }}
-        />
-        <Callout ref={this.calloutRef} />
-      </div>
-    );
+          )
+          : null
+        }
+        resultsVirtualize={false}
+        resultsOnMarkPosition={onMarkPosition}
+        resultsOnResetMarkedPosition={resetMarkedPosition}
+        resultsCachedPosition={itemToView}
+        nonInteractiveHeaders={['selected', 'jobParts']}
+      />
+      <ConfirmationModal
+        id="delete-selected-logs-modal"
+        open={showDeleteConfirmation}
+        heading={<FormattedMessage id="ui-data-import.modal.landing.delete.header" />}
+        message={(
+          <FormattedMessage
+            id="ui-data-import.modal.landing.delete.message"
+            values={{ logsNumber }}
+          />
+        )}
+        bodyTag="div"
+        confirmLabel={<FormattedMessage id="ui-data-import.modal.landing.delete" />}
+        cancelLabel={<FormattedMessage id="ui-data-import.modal.landing.cancel" />}
+        onConfirm={() => deleteLogs()}
+        onCancel={() => {
+          deselectAll();
+          hideDeleteConfirmation();
+        }}
+      />
+      <Callout ref={calloutRef} />
+    </div>
+  );
+};
+
+export const ViewAllLogsManifest = Object.freeze({
+  initializedFilterConfig: { initialValue: false },
+  query: {
+    initialValue: INITIAL_QUERY,
+  },
+  resultCount: { initialValue: INITIAL_RESULT_COUNT },
+  resultOffset: { initialValue: 0 },
+  records: {
+    type: 'okapi',
+    clear: true,
+    resultDensity: 'sparse',
+    resultOffset: '%{resultOffset}',
+    records: 'jobExecutions',
+    recordsRequired: '%{resultCount}',
+    path: getLogsPath,
+    params: getLogsQuery,
+    perRequest: RESULT_COUNT_INCREMENT,
+    throwErrors: false,
+    shouldRefresh: () => true,
+    resourceShouldRefresh: true,
+  },
+  usersList: {
+    type: 'okapi',
+    records: 'jobExecutionUsersInfo',
+    path: 'metadata-provider/jobExecutions/users',
+    throwErrors: false,
+    accumulate: true,
+    perRequest: USERS_LIMIT_PER_REQUEST,
+  },
+  jobProfilesList: {
+    type: 'okapi',
+    records: 'jobProfilesInfo',
+    path: 'metadata-provider/jobExecutions/jobProfiles',
+    throwErrors: false,
+    accumulate: true,
+    perRequest: JOB_PROFILES_LIMIT_PER_REQUEST,
+  },
+  splitStatus: {
+    type: 'okapi',
+    path: requestConfiguration,
+    throwErrors: false,
   }
-}
+});
+
+ViewAllLogs.manifest = ViewAllLogsManifest;
+
+ViewAllLogs.propTypes = {
+  mutator: PropTypes.object.isRequired,
+  resources: PropTypes.object.isRequired,
+  stripes: stripesShape.isRequired,
+  location: PropTypes.shape({
+    search: PropTypes.string.isRequired,
+    pathname: PropTypes.string.isRequired,
+  }).isRequired,
+  disableRecordCreation: PropTypes.bool,
+  browseOnly: PropTypes.bool,
+  packageInfo: PropTypes.object,
+  history: PropTypes.shape({ push: PropTypes.func.isRequired }),
+  refreshRemote: PropTypes.func,
+};
 
-export default ViewAllLogs;
+export default stripesConnect(ViewAllLogs);
```

Comment:
let's use `useStripes` instead of stripesConnect?

Reply:
It's vital wrapper in order to get `resources` from the `manifest`
E.g. (https://github.com/folio-org/ui-data-import/blob/master/src/settings/FileExtensions/EditFileExtension/EditFileExtension.js)